utils/config.go

```diff
@@ -43,6 +43,11 @@ type SSOSetting struct {
 	UserApiEndpoint string
 }
 
+type SSOSettings struct {
+	AllowEmailSignUp bool
```

Comment:
True, but that might be intended. I am imagining the admin console with a list of checkboxes for email setup:
- Email Signup
- Signup with google
- Signup with x...
If I want to disable signup I just uncheck all the boxes. 
I guess it just bothers me because signup with email is not SSO. But I am 0/5 on where the option is.